src/main/webapp/profile.js

```diff
@@ -14,14 +14,19 @@
 
 function onProfileLoad() {
   prepareFormValidation();
```

Comment:
Where is this method and why is it before supplyLogoutLink?

Reply:
lts in register.js